### PR TITLE
Add example for knowing what you have in your dataset

### PR DESCRIPTION
**Pull request by TheoBerglin**

### CODE REVIEW

File: docs-src/docs/dataset-exploration/understand-dataset-content.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Understand what your dataset contains
+---
+
+## Understand which scenes you've annotations for in your dataset
+
+To be able to know which scenes you can provide predictions for you need to understand which scenes you've
+annotations for in your dataset. This can be done by using an endpoint for listing annotations in your dataset. The following code snippet by utilizing an endpoint for
+will do this and save it to a csv file.
+
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/understand-dataset-content.md:10]: Maybe add that they currently can not add scenes to their dataset, if they think it is missing. Say that they should contact us in that case.

File: docs-src/docs/dataset-exploration/understand-dataset-content.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        done = True
+print(f"Fetched {len(annotations)} annotations")
+
+# save to csv called dataset_{dataset_uuid}_annotations.csv
+with open(f'dataset_{dataset_uuid}_annotations.csv', 'w', newline='') as csvfile:
+    writer = csv.writer(csvfile)
+    writer.writerow(annotations[0].keys())
+    for annotation in annotations:
+        writer.writerow(annotation.values())
+```
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/understand-dataset-content.md:39]: I haven't tried running this code myself, but I assume it works?

**Review by andreas-lindholm — APPROVED** [review]
Approve after comments

**Comment by TheoBerglin** [comment]
Thanks a lot for the review @andreas-lindholm 💯

**Pull request merged**